_includes/keyphrase.md

```diff
@@ -53,7 +53,7 @@ Use the following table to answer questions about keys/phrases/values you can sh
       </td>
    </tr>
    <tr valign="top">
-      <td><p>Stacks address or STX address</p>
+      <td><p>Stacks address or Stacks address</p>
```

Comment:
Why was this change made? `Stacks address or Stacks address` seems like a mistake. If possible, I'd not edit this file nor the glossary file.

Reply:
done as well